src/frontend/src/pages/myfeeds.js

```diff
@@ -14,6 +14,40 @@ const useStyles = makeStyles(theme => ({
 
 export default function MyFeeds() {
   const classes = useStyles();
+  const [feedAuthor, setFeedAuthor] = useState('');
+  const [feedUrl, setFeedUrl] = useState('');
+
+  function handleAuthorChange(author) {
+    setFeedAuthor(author);
+  }
+
+  function handleUrlChange(url) {
+    setFeedUrl(url);
+  }
+
+  async function addFeed() {
+    // if testing local dev use the first one, otherwise please use the second apiUrl
+    const telescopeUrl = 'http://localhost:3000';
+    // const telescopeUrl = 'dev.telescope.cdot.systems'
```

Comment:
[gatsby-config.js](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/gatsby-config.js#L19) defines `telescopeUrl` like so:

https://github.com/Seneca-CDOT/telescope/blob/6309e0defc732653bd0fe00d0274519ad25f6ba1/src/frontend/gatsby-config.js#L19
That is, after loading the `.env` file:
https://github.com/Seneca-CDOT/telescope/blob/6309e0defc732653bd0fe00d0274519ad25f6ba1/src/frontend/gatsby-config.js#L1-L8

Would doing the same make sense here?

Reply:
Yep, sorry. I don't know why I forgot about our process.env vars